composer.json

```diff
@@ -40,7 +40,6 @@
         "drupal/focal_point": "^1.5",
         "drupal/gin": "^3.0@alpha",
         "drupal/gin_toolbar": "^1.0@beta",
```

Comment:
Removed google_tag and it's config. This module should likely be added per project as it isn't always present.